vocabularies/trs.ttl

```diff
@@ -0,0 +1,178 @@
+
+This timeline of Parliaments of Australia starts in 1901 (Gregorian Caledar) when Australia was formed as an country from pre-existing colonies that where Dominions of the British Empire and were not sovereign states themselves. The 1901 Australian federal election for the inaugural Parliament of Australia was held in Australia on Friday 29 March and Saturday 30 March 1901 after which Edmond Barton was sworn in as the inaugural Prime Minister.
+
+Epoch within this timeline are numbered 1st, 2nd etc. up to (as of October, 2019, Gegorian Calendar) the 44th.""" ;
+  skos:prefLabel "Parliaments of Australia Timeline" ;
+  skos:topConceptOf trs:conceptScheme ;
+    skos:scopeNote "Use this timeline by quoting an epoch within it so, where in an XSD date using the Gregorian Calendar and 3 epochs (year, month, day) e.g. \"1982-11-05\" for the 11th of May, 1982, in Parliaments of Australia, the equivalent epoch (equivalent as in the one occuring on 11th of May, 1982) would simply be \"32\" which is a numerical version of \"32nd Parliament\" which started on the 25th of November, 1980 and ended on the 14–15the December, 1982. Since the start of this timescale in 1901, there have so far been 44 parliament epochs." ;
+.
+trs:SolarHijriCalendar
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Solar_Hijri_calendar> ;
+  skos:definition "The Solar Hijri calendar (Persian: گاه‌شماری هجری خورشیدی‎, romanized: gāh-shomāri-ye hejri-ye khorshidi; Pashto: لمريز لېږدیز کلیز‎), also called the Solar Hejri calendar or Shamsi Hijri calendar, and abbreviated as SH, is the official calendar of Iran and Afghanistan. It begins on the March equinox (Nowruz) as determined by astronomical calculation for the Iran Standard Time meridian (52.5°E, UTC+03:30) and has years of 365 or 366 days." ;
+  skos:prefLabel "Solar Hijri Calendar" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:UNIXTime
+  a skos:Concept ;
+  a time:TRS ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Unix_time> ;
+  skos:definition "Unix time (also known as Epoch time, POSIX time, seconds since the Epoch, or UNIX Epoch time) is a system for describing a point in time. It is the number of seconds that have elapsed since the Unix epoch, that is the time 00:00:00 UTC on 1 January 1970, minus leap seconds." ;
+  skos:prefLabel "UNIX Time" ;
+  skos:topConceptOf trs:conceptScheme ;
+.
+trs:UTCTime
+  a skos:Concept ;
+  a time:TRS ;
+  dct:source <https://www.bipm.org/en/bipm-services/timescales/> ;
+  rdfs:seeAlso <https://en.wikipedia.org/wiki/Unix_time> ;
+  skos:definition "The atomic time scale that forms the basis for the coordinated dissemination of standard frequencies and time signals" ;
```

Comment:
Ending full stop.

Reply:
added as per https://github.com/geological-survey-of-queensland/vocabularies/commit/c5c5bff5fa2e820aa68de308a74d97089b27a708